clang/lib/CodeGen/CGLoopInfo.h

```diff
@@ -111,11 +111,8 @@ struct LoopAttributes {
   /// Value for llvm.loop.ii.count metadata.
   unsigned SYCLIInterval;
 
-  /// Flag for llvm.loop.max_concurrency.count metadata.
-  bool SYCLMaxConcurrencyEnable;
-
   /// Value for llvm.loop.max_concurrency.count metadata.
-  unsigned SYCLMaxConcurrencyNThreads;
+  llvm::Optional<unsigned> SYCLMaxConcurrencyNThreads;
```

Comment:
Be sure to add the include to the top of the file for Optional (so you include what you use).

Reply:
> Be sure to add the include to the top of the file for Optional (so you include what you use).

I have added #include "llvm/ADT/Optional.h".